### DISCUSSION TITLE
INACTIVE Save pins buttons

### DISCUSSION THREAD

**Discussion by TintinSDev** [community/community#84628] [New to GitHub] [answered]
My save pins on customize pins per repository is inactive and I can't change the customized repos. Assist kindly

**Comment by levendlee** [comment]
Same here.

**Comment by darkotodoric** [comment, marked as answer]
Something broke on the GitHub side, probably they will fix this soon. Until that, click on "Customize your pins", press f12 on the page and enter this JS in the console:

```
document.querySelector('.js-pinned-items-submit').disabled = false
```

**Reply by Ashwin4514** [reply]
This worked. Thanks!

**Reply by TintinSDev** [reply]
They fixed it.Thanks

**Comment by akashdip2001** [comment]
If you're facing issues with saving pins or customizing them per repository on GitHub, you might want to try the following steps:

1. **Clear Browser Cache:**
   - Sometimes, browser cache issues can cause unexpected behavior. Clear your browser cache and try again.

2. **Browser Compatibility:**
   - Ensure that you are using a supported and updated browser. GitHub recommends using the latest versions of Google Chrome, Firefox, Safari, or Microsoft Edge.

3. **Disable Browser Extensions:**
   - Temporarily disable browser extensions, as they might interfere with GitHub's functionality. Try accessing GitHub in an incognito or private browsing window.

4. **Check GitHub Status:**
   - Verify if there are any ongoing issues or maintenance on GitHub by checking their status page: [GitHub Status](https://www.githubstatus.com/).

5. **Try a Different Browser:**
   - If the issue persists, try using a different browser to see if the problem is browser-specific.

6. **GitHub Desktop:**
   - If you are using GitHub Desktop, check if the issue persists when accessing GitHub directly in a web browser.

7. **Update GitHub Desktop:**
   - If you are using GitHub Desktop, make sure you have the latest version installed.

8. **Contact GitHub Support:**
   - If none of the above steps resolves the issue, consider reaching out to GitHub Support for assistance. You can submit a support request through the GitHub Help form: [GitHub Contact Support](https://support.github.com/contact).

By following these steps, you should be able to troubleshoot and resolve the issue with saving and customizing pins on GitHub.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by ghostinhershell** [reply]
Hi @akashdip2001, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by akashdip2001** [reply]
Sorry ma'am, in future I definitely consider the guidelines of GitHub

Thank you :)
